robottelo/test.py

```diff
@@ -171,6 +171,19 @@ class CLITestCase(TestCase):
     """Test case for CLI tests."""
     _multiprocess_can_split_ = True
 
+    def assert_error_msg(self, raise_ctx, *contents):
+        """
```

Comment:
nitpick, but robottelo code style start docstrings in the above line

instead of

```python
def name(...):
    """
    first line
    second line
    """
```

we do

```python
def name(...):
    """first line
    second line
    """
```

important for documentaion generation and getting help on console.